transcendence-app/src/user/infrastructure/db/postgres/block.postgres.repository.ts

```diff
@@ -31,4 +31,18 @@ export class BlockPostgresRepository
     });
     return block && block.length ? block[0] : null;
   }
+
+  async deleteBlock(
+    blockerId: string,
+    blockedId: string,
+  ): Promise<BlockEntity | null> {
+    const block = await makeQuery<BlockEntity>(this.pool, {
+      text: `DELETE
+      FROM ${this.table}
+      WHERE ${BlockKeys.BLOCKER_ID} = $1 AND ${BlockKeys.BLOCKED_ID} = $2
+      RETURNING *;`,
```

Comment:
what does the statement RETURNING in this context do?

Reply:
By using [`RETURNING *`](https://www.postgresql.org/docs/15/dml-returning.html), Postgres returns the content of the deleted rows.
We could also use the [`rowcount`](https://node-postgres.com/api/result#resultrowcount-int) variable from the `pg.Result`.